tasks/optional-label-check.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+
+  steps:
+    - name: inspect-image
+      image: quay.io/rsaar/hacbs-test:latest
+      script: "skopeo inspect docker://$(inputs.params.sourceImage) > /tekton/home/image_inspect.json"
```

Comment:
I wonder if we should output to a different named file like `tekton/home/optional_label_image_inspect.json` or a quick bash if file exists check don't run the skopeo command. That way if we want to POC parallelizing our sanity tasks we can avoid a race condition with skopeo until hacbs-220 is complete? Thoughts @staticf0x @hongweiliu17 @jsztuka @dirgim

Reply:
Checking if file exists is a good point. let me add to my hacbs-220

Reply:
I changed the file name, good point @Dannyb48